Use new version of PeerNet with the `stream_limiter` set

litchipi opened the pull request:
Uses the changes from https://github.com/massalabs/PeerNet/pull/55 inside the `massa_protocol_{worker,exports}`

* [ ] document all added functions
* [ ] try in sandbox /simulation/labnet
* [ ] unit tests on the added/changed features
  * [ ] make tests compile
  * [ ] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [ ] if the API has changed, update the API specification

litchipi commented on massa-node/src/operation_injector.rs:
```diff
@@ -113,7 +114,7 @@ pub fn start_operation_injector(
                 .propagate_operations(storage.clone())
                 .unwrap();
-            std::thread::sleep(Duration::from_secs(1));
+            std::thread::sleep(Duration::from_secs(1).saturating_sub(now.elapsed()));
```
This is a fix for the operation injector, I'll take it out of this PR and create a new one especially for this

AurelienFT replied:
You can let it there it's ok

litchipi replied:
PR: #4194

Leo-Besancon reviewed (approved):
Congrats!

litchipi commented:
I just fixed tests in `massa-bootstrap`. Because some `Limiter` were created in advance, they had the time to stack a bit of tokens before the read, so it took less time than it was expected.
To solve this, I added a `std::thread::sleep` to stack to the maximum the Limiters, then assert they take longer than expected (but lower the expectation as we stacked up)
